app/services/container_dashboard_service.rb

```diff
@@ -0,0 +1,206 @@
+        :count     => @ems.present? ? @ems.container_projects.count : ContainerProject.count
+      },
+      {
+        :name      => "Pods",
+        :iconClass => "fa fa-cubes",
+        :count     => @ems.present? ? @ems.container_groups.count : ContainerGroup.count,
+        :status    => [
+          {
+            :iconClass => "pficon-error-circle-o",
+            :count     => 3 #dummy data
+          }
+        ]
+      },
+      {
+        :name      => "Services",
+        :iconClass => "pficon-service",
+        :count     => @ems.present? ? @ems.container_services.count : ContainerService.count
+      },
+      {
+        :name      => "Images",
+        :iconClass => "pficon-image",
+        :count     => @ems.present? ? @ems.container_images.count : ContainerImage.count
+      },
+      {
+        :name      => "Routes",
+        :iconClass => "pficon-route",
+        :count     => @ems.present? && @ems.kind_of?(ManageIQ::Providers::Openshift::ContainerManager) ?
+                      @ems.container_routes.count :
+                      ContainerRoute.count
+      }
+    ]
+  end
+
+  def providers
+    # Change this for when @ems.present?
+    {
+      :title         => "Providers",
+      :count         => ManageIQ::Providers::Kubernetes::ContainerManager.count +
+                        ManageIQ::Providers::Openshift::ContainerManager.count,
+      :notifications => [
+        {
+          :iconClass => "pficon-openshift",
+          :count     => ManageIQ::Providers::Openshift::ContainerManager.count,
+        },
+        {
+          :iconClass => "pficon-kubernetes",
+          :count     => ManageIQ::Providers::Kubernetes::ContainerManager.count,
+        }
+      ]
+    }
+  end
+
+  def node_utilization
+    cpu_total = 0
+    mem_total = 0
+
+    if @ems.present?
+      providers = [@ems]
+    else
+      providers = (ManageIQ::Providers::Openshift::ContainerManager.all <<
+                     ManageIQ::Providers::Kubernetes::ContainerManager.all).flatten
+    end
+
+    providers.each do |prov|
+      cpu_total += prov.aggregate_logical_cpus # get this from metrics as well?
+      mem_total += prov.aggregate_memory       # get this from metrics as well?
+    end
+
+    mem_total = memory_to_human_size(mem_total).to_i # Always in GB?
+
+    {
+      :cpu => {
+        :total => cpu_total,
+        :used  => cpu_total - 3, # Dummy data
+        :xData => (["dates"] << (1..30).to_a).flatten, # Dummy data
+        :yData => (["used"] << (1..30).collect { |_x| rand(cpu_total) }).flatten # Dummy data
+      },
+      :memory => {
+        :total => mem_total,
+        :used  => mem_total - 7, # Dummy data
+        :xData => (["dates"] << (1..30).to_a).flatten, # Dummy data
+        :yData => (["used"] << (1..30).collect { |_x| rand(mem_total) }).flatten # Dummy data
+      }
+    }
+  end
+
+  def memory_to_human_size(mem)
+    ActionView::Base.new.number_to_human_size(mem * 1.megabyte, :precision => 0).split[0]
```

Comment:
Oh, no, I was thinking of config/initializers/av_number_helper.rb

But even so, this feels wrong because it just takes the number without the unit.  The unit can change, thus the meaning of the number changes.

```ruby
[1] pry(main)> mem = 6144 # megabytes
=> 6144
[2] pry(main)> ActionView::Base.new.number_to_human_size(mem * 1.megabyte, :precision => 0).split[0]
=> "6"
[3] pry(main)> mem = 100 # megabytes
=> 100
[4] pry(main)> ActionView::Base.new.number_to_human_size(mem * 1.megabyte, :precision => 0).split[0]
=> "100"
```